extensions/security/spi/src/main/java/io/quarkus/security/spi/AdditionalSecuredClassesBuildItem.java

```diff
@@ -1,30 +0,0 @@
-package io.quarkus.security.spi;
-
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
-
-import org.jboss.jandex.ClassInfo;
-
-import io.quarkus.builder.item.MultiBuildItem;
-
-/**
- * Contains classes that need to have @DenyAll on all methods that don't have security annotations
- */
-public final class AdditionalSecuredClassesBuildItem extends MultiBuildItem {
```

Comment:
We can't really delete SPI build items (as other extensions could be using them), but we can deprecate them while still maintaining their functionality

Reply:
Okay, will do.

Reply:
Addressed, thanks.